langkit/themes.py

```diff
@@ -70,7 +70,7 @@ def refusal_similarity(text: str) -> Optional[float]:
 def load_themes(json_path: str):
     try:
         skip = False
-        with open(json_path, "r") as myfile:
+        with open(json_path, "r", encoding="utf-8") as myfile:
```

Comment:
wouldn't it be best if it were an optional argument to `load_themes` that defaults to "utf-8" instead?

Reply:
that's better! done! thanks!